### ISSUE TITLE
Audio drops when performing beatjump

### ISSUE THREAD

**Issue by acolombier** [mixxxdj/mixxx#13159]
### Bug Description

When performing large beatjumps (8+ beats), it happens sometimes that there is audio drops. I believe this is due to CachingReader miss, leading to delay to fetch the samples.
A suggestion would be to add Beatjump as a `Hint::Type` at `playposition+beatjump size` and `playposition-beatjump size`

### Version

main

### OS

Pop OS 22.04 (with Wayland)

**Comment by ronso0** [comment]
So you suggest to hint the reader with `Hint::Type::Beatjump` in `LoopingControl::hintReader`?
If I understand the flow correctly, this requires tracking the beatjump seek request or get it from EngineBuffer?

**Comment by acolombier** [comment]
Yes, that's my suggestion, and from my understanding so far, this would indeed require tracking the beatjump seek request to add a hint at -/+ the current play position.

**Comment by cokeramirez** [comment]
is this the same that #8842?

I experiment this too, I notice that if you jump forward and then inmediatly back there is no gap ussually

**Comment by acolombier** [comment]
Ah good catch! Indeed this is the same issue.
